tests/cli/ursula/test_stakeholder_and_ursula.py

```diff
@@ -375,7 +375,9 @@ def test_collect_rewards_integration(click_runner,
     blockchain_alice.selection_buffer = 1
 
     M, N = 1, 1
-    expiration = maya.now() + datetime.timedelta(days=3)
+    days = 3
+    now = testerchain.w3.eth.getBlock(block_identifier='latest').timestamp
+    expiration = maya.MayaDT(now).add(days=days-1)
```

Comment:
I don't know if I fully grasp this - we use `days-1` as opposed to `days` because the current day counts as one i.e. is included?

Reply:
Days here means how many days must be paid, duration could be one period from noon to noon but need to pay for two periods.

Reply:
👍